backend/test/test_github_app_client.py

```
@@ -0,0 +1,159 @@
+import httpx
+import pytest
+from unittest.mock import AsyncMock, MagicMock, patch
+
+from github.github_app_client import (
+    generate_jwt,
+    get_installation_access_token,
+)
+
+class DummyResponse:
+	def __init__(self, data=None, exc=None):
```

Ruff (0.15.11)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)